options/locale/locale_en-US.ini

```diff
@@ -681,6 +681,7 @@ stored_lfs = Stored with Git LFS
 commit_graph = Commit Graph
 blame = Blame
 normal_view = Normal View
+lines = lines
```

Comment:
Not that I mind, but this doesn't cover the "1 line" case. Otherwise, LG-TM.

Reply:
Indeed, will probably do a followup on that.

Reply:
@silverwind I already created PR for that (already merged ;) )